includes/Story_Post_Type.php

```diff
@@ -693,34 +729,34 @@ public static function filter_template_include( $template ) {
 	 *
 	 * @return string Publisher logo image URL. WordPress logo if no site icon or custom logo defined, and no logo provided via 'amp_site_icon_url' filter.
 	 */
-	private static function get_publisher_logo() {
+	public static function get_publisher_logo() {
 		$logo_image_url = null;
 
-		// This should be square, at least 96px in width/height. The 512 is used because the site icon would have this size generated.
-		$logo_width  = 512;
-		$logo_height = 512;
-
-		// Use the Custom Logo if set, but only if it is square.
-		$custom_logo_id = get_theme_mod( 'custom_logo' );
-		if ( has_custom_logo() && $custom_logo_id ) {
-			$custom_logo_img = wp_get_attachment_image_src( $custom_logo_id, [ $logo_width, $logo_height ], false );
-			if ( $custom_logo_img && ( $custom_logo_img[2] === $custom_logo_img[1] ) ) {
-				$logo_image_url = $custom_logo_img[0];
+		$publisher_logo_settings = get_option( 'web_stories_publisher_settings', [] );
+		$has_publisher_logo      = ! empty( $publisher_logo_settings['active'] );
```

Comment:
So will this reactivity change on older stories? What if you want different logo for different times of the year, like a christmas / pride logo or something. Is that the point of multiple logos.

Reply:
> What if you want different logo for different times of the year, like a christmas / pride logo or something. Is that the point of multiple logos.

I guess so! Or perhaps each story can have their own publisher logo, e.g. a Story about Christmas has a different publisher logo than the others 🤷‍♀ 

The behavior of publisher logos should be addressed in a separate issue, likely in #570.

Reply:
Why isnt this be tackelled here?